src/hono.ts

```diff
@@ -7,6 +7,8 @@ import { METHOD_NAME_ALL, METHOD_NAME_ALL_LOWERCASE } from './router'
 import { TrieRouter } from './router/trie-router' // Default Router
 import { getPathFromURL, mergePath } from './utils/url'
 
+export interface ContextVariableMap {}
```

Comment:
I forget the specifics, but this should probably extend `Record<string, any>`

Reply:
In this PR, there is no need to extend `Record<string, any>` since `ContextVariableMap` is a type for type definition and no object is created. (This is like [HTMLElementTagNameMap](https://github.com/microsoft/TypeScript/blob/949fffb1e0b683a3b4fceb915e7fd4ddb4b54798/lib/lib.dom.d.ts#L18029-L18147).)
Also, not extending `Record<string, any>` will give better completion results.